pages/3.0/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Jakarta Server Pages 3.0 (milestone)"
+title: "Jakarta Server Pages 3.0"
 date: 2020-06-13
```

Comment:
This date will need to be changed at some point.  When you get closer to a ballot, you can put in a more reasonable date.

Reply:
Updated date to today, we can update again if necessary.